packages/actor-init-sparql/test/HttpServiceSparqlEndpoint-test.ts

```diff
@@ -761,3 +686,360 @@ describe('HttpServiceSparqlEndpoint', () => {
     });
   });
 });
+
+describe('A second HttpServiceSparqlEndpoint instance', () => {
```

Comment:
Is there a reason why you decided to move these tests into a new describe block with a second instance?